### ISSUE TITLE
Every folder i open have " - File Explorer" added to its name

### ISSUE THREAD

**Issue by romafrogs** [valinet/ExplorerPatcher#3792]
Win 11

any way to remove that annoying text?

**Comment by pyrates999** [comment]
You're running an insider build of windows and that was recently added to it.  Blame MS for that.

**Comment by romafrogs** [comment]
Not sure you are right, but ill try to explain the issue:

When you open folders and minimize them to the task bar 
" - File Explorer" is added to the title of each tab

**Comment by pyrates999** [comment]
Yes, someone else mentioned that as a comment on here.  MS added that.

**Comment by romafrogs** [comment]
Do you have a link?
when im contacting MS they have no idea why is that taking palace

**Comment by pyrates999** [comment]
I do not.  I just remember it being left as a comment in another issue here.

what if you uninstall EP?

**Comment by zhanxiaoge** [comment]
After updating my system and EP today, I encountered the same issue. I have not joined the Windows Insider Preview program. My OS version is 22631.4317, and the EP version is 22621.3880.66.6.

**Comment by romafrogs** [comment]
me too - super annoying

**Comment by Amrsatrio** [comment]
So you want EP to remove the "- File Explorer" which latest Windows updates brought?

**Comment by zhanxiaoge** [comment]
> So you want EP to remove the "- File Explorer" which latest Windows updates brought?

YES! We also don't know if Windows 11 will fix this issue.

**Comment by pyrates999** [comment]
It might be on purpose to let you know it is file explorer that you have open.

**Comment by romafrogs** [comment]
its not on purpose. i have spoke with MS via chat - they ara clueless

**Comment by pyrates999** [comment]
Do other applications, that you have open, include "- application name" on their application icon in the taskbar when opened?

**Comment by romafrogs** [comment]
no. its only for folders

**Comment by Masfo** [comment]
22H2 (22621.4317), I find it very distracting as it just repeats for all folder. It would be a nice feature to remove it and the **" and N more tab(s)"** when you have tabs open.

![image](https://github.com/user-attachments/assets/b6de3e7f-5fad-42ff-986b-ed920587879f)

**Comment by 01x2be3** [comment]
Please please make a feature to remove it! Its just so useless and annoying thing

**Comment by Zubastic** [comment]
Uninstall update KB5044285. This "security" update blows up explorer title.

**Comment by romafrogs** [comment]
you cant uninstall that update

**Comment by Zubastic** [comment]
> you cant uninstall that update

https://support.microsoft.com/en-us/windows/how-to-uninstall-a-windows-update-c77b8f9b-e4dc-4e9f-a803-fdec12e59fb0

**Comment by romafrogs** [comment]
I know to remove install updates.
Im saying you CANT uninstall that specific one

**Comment by Zubastic** [comment]
> Im saying you CANT uninstall that specific one

I already DID IT.

**Comment by romafrogs** [comment]
on win 11?

**Comment by romafrogs** [comment]
it goes into the same place after all.
I can only uninstall 5 updates
KB5044285 isnt on the list

**Comment by explor202** [comment]
> So you want EP to remove the "- File Explorer" which latest Windows updates brought?

I would love to see this option too!

**Comment by explor202** [comment]
I've been able to remove "file explorer" from being displayed using a Windhawk mod, but it's only working on 2 23H2 PC's. I've been unsuccessful getting it to work on 24H2, but I'm not sure if that's a compatibility issue or something else. I would much prefer having this handled by EP though!

**Comment by pyrates999** [comment]
Can you name the windhawk mod that removes the text `file explorer` from file explorer windows?  I think EP only needs to update it's reimplemented windows 10 taskbar to support it then.

**Comment by explor202** [comment]
I'm using the Aerexplorer mod on 2 23H2 PC's and it works well at removing the suffix, though it does seem to increase the spacing between folder names. The mod does not appear to be compatible with 24H2. 

The ability to remove the suffix, adjust spacing (compact/not compact) and increase/decrease the font size would be ideal.

**Comment by pyrates999** [comment]
Thank you.  Please wait for a developer to respond.

**Comment by PantheraTigris1** [comment]
I believe this can be fixed by editing explorerframe.dll.mui which is located in \System32\en-GB

**Comment by albino1** [comment]
FWIW, there's now a Windhawk mod for this that works with Windows 11 24H2:

https://windhawk.net/mods/file-explorer-remove-suffixes

**Comment by pyrates999** [comment]
yes, in order for windhawk mods to work, windhawk needs to be constantly running.

**Comment by romafrogs** [comment]
I wished for a solution that just works one time for good
without any app working in the background
